From 60c9649059980e9e96af755b69e888d8bf1e2b9f Mon Sep 17 00:00:00 2001
From: Dr Nic Williams <drnicwilliams@gmail.com>
Date: Fri, 19 Dec 2014 22:18:13 -0800
Subject: [PATCH] clean bundler env before calling out to bosh-registry

---
 lib/bosh/deployer/registry.rb | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/lib/bosh/deployer/registry.rb b/lib/bosh/deployer/registry.rb
index 6da48eb..57c51d8 100644
--- a/lib/bosh/deployer/registry.rb
+++ b/lib/bosh/deployer/registry.rb
@@ -30,7 +30,9 @@ module Bosh::Deployer
 
       cmd = "bosh-registry -c #{@registry_config.path}"
 
-      @registry_pid = Process.spawn(cmd)
+      Bundler.with_clean_env {
+        @registry_pid = Process.spawn(cmd)
+      }
 
       watch_for_crash(cmd)
       wait_for_listen
-- 
2.2.1

